sdk/iOS/OoyalaSkinSDK/ViewControllers/FullscreenStateController/FullscreenStateController.m

```diff
@@ -31,6 +29,10 @@ @interface FullscreenStateController()
 
 @implementation FullscreenStateController
 
+#pragma mark - Constants
+
+static double const FULLSCREEN_ANIMATION_DURATION = 0.4;
```

Comment:
is it 400 millis? why 400?

Reply:
yes, this value in seconds. Its time period selected, because it's great time combination with iOS rotated animation. We experimented with other values, but this is the best value.